bin/configs/csharp-restsharp-net7.0-useDateTimeForDate.yaml

```diff
@@ -0,0 +1,9 @@
+# for .net standard
+generatorName: csharp
+outputDir: samples/client/petstore/csharp/OpenAPIClient-net7.0-useDateTimeForDate
```

Comment:
in https://github.com/OpenAPITools/openapi-generator/tree/master/.github/workflows, there are workflows for dotnet.

can you please add the new sample folders to these workflow so that CI will tests to make sure these samples compile without issues moving forward?

Reply:
I've also noticed that *netcore-latest* samples not captured in `on` section's regexes. Is it intended or mistake?
![image](https://github.com/OpenAPITools/openapi-generator/assets/17771546/dfaf7d56-2aaa-446a-8d96-f08b1bcf676b)

Reply:
likely a mistake. can you please file another PR to fix it?

Reply:
Sure.